src/downloadDocumentation.spec.js

```diff
@@ -59,9 +59,10 @@ describe('ServerlessAWSDocumentation', function () {
       });
     });
 
-    it('should successfully download documentation, yaml extension', async () => {
+    it.each([['yml'], ['yaml']])('should successfully download documentation, %s extension', async (outputFileExtension) => {
+      const outputFileName = `test.${outputFileExtension}`;
       objectUnderTest.options = {
-        outputFileName: 'test.yml',
+        outputFileName: outputFileName,
```

Comment:
This change is totally unrelated. But, as I was touching this test suite, I thought it useful to add coverage also for this already existing use case (i.e., using a `yaml` file extension).
If you prefer not to, let me know and I will revert it.